src/site/content/en/metrics/inp/index.md

```diff
@@ -196,7 +250,7 @@ As with other methods exported by `web-vitals`, `onINP` accepts a function as an
 See the [`onINP()`](https://github.com/GoogleChrome/web-vitals#oninp) reference documentation for additional usage instructions.
 
 {% Aside 'warning' %}
-Gathering INP metrics in the field will only work on browsers that [fully support the Event Timing API](https://caniuse.com/mdn-api_performanceeventtiming), including its `interactionId` property.
+Gathering INP metrics in the field will only work on browsers that fully support the Event Timing API, including its `interactionId` property.
```

Comment:
```suggestion
Gathering INP metrics in the field will only work on browsers that [fully support the Event Timing API, including its `interactionId` property](https://developer.mozilla.org/docs/Web/API/PerformanceEventTiming/interactionId#browser_compatibility).
```

Reply:
I removed this link since it was linked earlier, but can commit if you feel strongly about this.

Reply:
I changed the link specifically to show which browsers support the `interactionId` property. So not same link as previously. That's also why I expended the link text.